lib/plugins/linkedin.py

```diff
@@ -47,16 +49,24 @@ def wrap_text_with_index(self, content):
         return final_lines
 
     def format_content(self, content, mentions, hashtags, images, **kwargs):
+        # the mentions are not linked to anyone!
         mentions = " ".join([f"@{v}" for v in mentions])
-        hashtags = " ".join(
-            [f"#{v}" for v in hashtags]
-        )  # the mentions are not linked to anyone!
+        hashtags = " ".join([f"#{v}" for v in hashtags])
         if len(images) > 20:
             warnings = f"A maximum of 20 images, not {len(images)}, can be included in a single linkedin post."
             images = images[:20]
         else:
             warnings = ""
 
+        # convert markdown formatting because Mastodon doesn't support it
```

Comment:
should this be a shared function in both classes?

Reply:
Should I create a function in `galaxy_social.py` to be used by plugins?